src/components/Booklist.js

```diff
@@ -1,21 +1,29 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
-import Addbook from './Addbook';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import CreateBook from './Books';
+import { getBooks } from '../redux/books/bookSlice';
 
 const Booklist = () => {
-  const { books } = useSelector((state) => state.books);
+  const books = useSelector((state) => state.books.books);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getBooks());
+  }, [dispatch]);
+
   return (
     <div>
       {books.map((book) => (
-        <CreateBook
-          key={book.item_id}
-          title={book.title}
-          author={book.author}
-          id={book.item_id}
-        />
+        <li key={book.item_id}>
+          <CreateBook
+            title={book.title}
+            author={book.author}
+            id={book.item_id}
+
+          />
+        </li>
```

Comment:
- [ ] As suggested by the previous reviewer, I would kindly advise you to remove the list-style-type dot showing in your design as this makes your design looks a little bit different from the original zeplin design.
You can set the list-style-type property to `none`

Your Design 👇 

<img width="953" alt="image" src="https://github.com/Roland-Ntwali/Bookstore/assets/78634841/41fe9b21-7a86-4bb6-a370-c3cbb05f75bf">